web/src/app/users/UserContactMethodEditDialog.js

```diff
@@ -57,7 +56,7 @@ export default function UserContactMethodEditDialog({
           <UserContactMethodForm
             errors={fieldErrs}
             disabled={loading}
-            edit={edit}
+            edit={true}
```

Comment:
whoops, sorry for that suggestion, with the linter it should read as
```suggestion
            edit
```